robocluster/device.py

```diff
@@ -57,6 +58,8 @@ def __init__(self, name, group, loop=None):
         )
         self._receiver.bind()
 
+        self._serial_device = None
```

Comment:
Is there anything preventing us from having multiple serial devices on the same `Device`?

Reply:
I don't think so. I probably just wanted to get one working first, but we should be able to support multiple.